managed/CounterStrikeSharp.API/Modules/Menu/ConsoleMenu.cs

```diff
@@ -43,18 +43,20 @@ public override void Display()
                 var option = Menu.MenuOptions[i];
 
                 Player.PrintToConsole(
-                    $" {(option.Disabled ? "[Enabled]" : "[Disabled] - ")} !{keyOffset++} {option.Text}");
+                    $"{(option.Disabled ? "[DISABLE]" : "[ENABLE]")}[!{keyOffset++}] {option.Text}");
             }
-            
+
             if (HasPrevButton)
             {
-                Player.PrintToConsole($"!7 -> Prev");
+                Player.PrintToConsole("[!7] -> Prev");
             }
 
             if (HasNextButton)
             {
-                Player.PrintToConsole($"!8 -> Next");
+                Player.PrintToConsole("[!8] -> Next");
             }
+
+            Player.PrintToConsole("[!9] -> Exit");
```

Comment:
Rather than `!9` etc for the console menu we should probably prefix the options with `css_{KEY}` instead. No square brackets required.

Reply:
Agreed